docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,43 @@
 title: Changelog
 ---
 
+## 12.9.0
+
+_Released 03/28/2023_
+
+**Features:**
+
+- The [Debug page](https://docs.cypress.io/guides/cloud/runs#Debug) now allows
+  for navigating between all runs recorded for a commit. Addresses
+  [#25899](https://github.com/cypress-io/cypress/issues/25899) and
+  [#26018](https://github.com/cypress-io/cypress/issues/26018).
+
+**Bugfixes:**
+
+- Fixed a compatibility issue so that component test projects can use
+  [Vite](https://vitejs.dev/) version 4.2.0 and greater. Fixes
+  [#26138](https://github.com/cypress-io/cypress/issues/26138).
+- Fixed an issue where
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) added an
+  additional `content-length` header to spied requests that did not set a
+  `content-length` header on the original request. Fixes
+  [#24407](https://github.com/cypress-io/cypress/issues/24407).
+- Changed the way that Git hashes are loaded so that non-relevant runs are
+  excluded from the Debug page. Fixes
+  [#26058](https://github.com/cypress-io/cypress/issues/26058).
+- Corrected the [`.type()`](https://docs.cypress.io/api/commands/type) command
```

Comment:
Here and above, `docs.cypress.io` links should be relative

Reply:
I'll get it right one of these days 😅 . fixed in 99a8a2aa4